crates/biome_js_formatter/src/utils/member_chain/simple_argument.rs

```diff
@@ -161,6 +163,19 @@ impl SimpleArgument {
         }
     }
 
+    fn is_simple_regex_expression(&self) -> bool {
```

Comment:
Let's add a documentation comment that explains **when** a regex is simple.

While you added this information to the PR, that comment will be lost; instead, having it as a comment is helpful to future contributors.